Add a tip about needing server and client usage in intra-cluster TLS certificates (#1688)
neo4j/docs-operations: modules/ROOT/pages/clustering/setup/encryption.adoc

It generally requires having a PKI with a xref:security/ssl-framework.adoc#term-ssl-certificate-authority[Certificate Authority (CA)] within the organization and they should be able to advise here.
Note that the information in this manual relating to the PKI is mainly for illustrative purposes.

[TIP]
====

> If setting up intra-cluster encryption as part of a cluster configuration, do ensure that the certificates used on the cluster endpoint supports both Server and Client usage.

Reviewer: ```suggestion
If setting up intra-cluster encryption as part of a cluster configuration, ensure that the certificates used on the cluster endpoint support server and client usage.
```

> This is because when connecting between the Neo4j servers for clustering, each server will use its own certificate to authenticate as a client on the connection to another server.

Reviewer: ```suggestion
This is because when connecting between the Neo4j servers for clustering, each server uses its own certificate to authenticate as a client on the connection to another server.
```

This could be verified from within the certificate details:

> ```

Reviewer: ```suggestion
----
```

openssl x509 -in public.crt -noout -text

> ```

Reviewer: ```suggestion
----
```

We should see that the X509v3 Extended Key Usage section shows both the usages listed:

> ```

Reviewer: ```suggestion
----
```

X509v3 Extended Key Usage:
    TLS Web Server Authentication, TLS Web Client Authentication

> ```

Reviewer: ```suggestion
----
```
